<?xml version="1.0"?>
<launch>

    <arg name="id_dict" default="$(find multi_car_localization)/param/ids.yaml"/>
    <arg name="num_cars" default="4" />

    <arg name="connections" default="$(find multi_car_localization)/param/connections_four_cars.yaml"/>

    <rosparam command="load" file="$(arg id_dict)"/>
    <rosparam command="load" file="$(arg connections)"/>

    <include file="$(find multi_car_localization)/launch/fake_uwb.launch">
        <arg name="num_cars" value="$(arg num_cars)" />
    </include>

    <include file="$(find multi_car_localization)/launch/fake_car.launch" ns="car0">
        <arg name="car_frame_id" value="car0" />
        <arg name="num_cars" value="$(arg num_cars)" />
        <arg name="dynamics_model" value="dubins_offset" />
    </include>

    <include file="$(find multi_car_localization)/launch/fake_car.launch" ns="car1">
        <arg name="car_frame_id" value="car1" />
        <arg name="num_cars" value="$(arg num_cars)" />
        <arg name="dynamics_model" value="dubins_offset" />
    </include>

    <include file="$(find multi_car_localization)/launch/fake_car.launch" ns="car2">
        <arg name="car_frame_id" value="car2" />
        <arg name="num_cars" value="$(arg num_cars)" />
        <arg name="dynamics_model" value="dubins_offset" />
    </include>

    <include file="$(find multi_car_localization)/launch/fake_car.launch" ns="car3">
        <arg name="car_frame_id" value="car3" />
        <arg name="num_cars" value="$(arg num_cars)" />
        <arg name="dynamics_model" value="dubins_offset" />
    </include>

    
</launch>